libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Analytics/SFSDKEventBuilderHelper.m

```diff
@@ -45,7 +48,12 @@ + (void) createAndStoreEvent:(NSString *) name userAccount:(SFUserAccount *) use
     SFSDKSalesforceAnalyticsManager *manager = [SFSDKSalesforceAnalyticsManager sharedInstanceWithUser:account];
     SFSDKInstrumentationEvent *event = [SFSDKInstrumentationEventBuilder buildEventWithBuilderBlock:^(SFSDKInstrumentationEventBuilder *builder) {
         builder.name = name;
-        builder.startTime = [[NSDate date] timeIntervalSince1970];
+        if (attributes[kSFSDKEventBuilderHelperStartTime]) {
```

Comment:
buildEvent will use current time if start time is 0, so no need to specify it here.
(Furthermore it should have been `[[NSDate date] timeIntervalSince1970] * 1000` since milliseconds are expected).